airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/manifest_component_transformer.py

```diff
@@ -57,18 +57,18 @@
     "SubstreamSlicer.parent_stream_configs": "ParentStreamConfig",
     # AddFields
     "AddFields.fields": "AddedFieldDefinition",
-    # CustomStreamSlicer
-    "CustomStreamSlicer.end_datetime": "MinMaxDatetime",
-    "CustomStreamSlicer.end_time_option": "RequestOption",
-    "CustomStreamSlicer.parent_stream_configs": "ParentStreamConfig",
-    "CustomStreamSlicer.start_datetime": "MinMaxDatetime",
-    "CustomStreamSlicer.start_time_option": "RequestOption",
+    # CustomPartitionRouter
+    "CustomPartitionRouter.end_datetime": "MinMaxDatetime",
+    "CustomPartitionRouter.end_time_option": "RequestOption",
+    "CustomPartitionRouter.parent_stream_configs": "ParentStreamConfig",
+    "CustomPartitionRouter.start_datetime": "MinMaxDatetime",
+    "CustomPartitionRouter.start_time_option": "RequestOption",
```

Comment:
For `end_datetime`, `end_time_option`, `start_datetime`, and `start_time_option`, we can instead make these defaults for the `CustomIncrementalSync` component since they are properties of the `DatetimeBasedCursor` where they might need to be injected